lib/tasks/utils.js

```diff
@@ -0,0 +1,84 @@
+'use strict';
+
+// Include Gulp, plugins and local files
+var fs = require('fs');
+var handlebars = require('handlebars');
+var mkdirp = require('mkdirp');
+
+// Define Base folders 
+var srcDir = {
```

Comment:
This `srcDir` object is very specific to the build process and should be in `build.js`